CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.8.2.2
+=======
+
+* [#297](https://github.com/serokell/universum/pull/297)
+  * User-invisible change to support GHC-9.10.
```

Comment:
Is "user-invisible" common terminology? I'd just say "Add support for GHC 9.10"

Reply:
I slightly dislike "Add support for GHC 9.10" because sometimes it hides user-visible changes which can matter to whoever reads this changelog. I think I'll change it to "Add support for GHC 9.10 without any user-visible changes".

P. S. You may argue that the fact that I'm changing only the latest version component already implies that there are no user-visible changes, but sometimes people don't follow PVP (intentionally or not), so it's not guaranteed.